react-components/src/components/Architecture/pointsOfInterest/PoiList.tsx

```diff
@@ -1,44 +1,61 @@
 /*!
  * Copyright 2024 Cognite AS
  */
-import { type ReactNode, useMemo, useState } from 'react';
+import { type ReactNode, useCallback, useMemo, useState } from 'react';
 import { type PointOfInterest } from '../../../architecture/concrete/pointsOfInterest/types';
 import { useOnUpdateDomainObject } from '../useOnUpdate';
 import { DataGrid, type DatagridColumn } from '@cognite/cogs-lab';
 import { usePoiDomainObject } from './usePoiDomainObject';
 import { EMPTY_ARRAY } from '../../../utilities/constants';
 import { useTranslation } from '../../i18n/I18n';
+import { Button, Flex } from '@cognite/cogs.js';
+import { take } from 'lodash';
 
 type RowType = {
   id: string;
   poi: PointOfInterest<unknown>;
 };
 
+export type PoiFilter = { contains: string };
```

Comment:
Should the type be exported?

Reply:
I think it's the "right" thing to do, since it's part of the function signature